### PR TITLE
Feat :  add whitepaper v2 solution

### PR DESCRIPTION
**Pull request by MathieuConte**
New branch `feat/whitepaper-v2-solution`, built from `feat/whitepaper-v2-introduction`

- New section 2 added

- draft images added

### CODE REVIEW

File: docs/whitepaper/solution.md [hunk 1]
```diff
@@ -0,0 +1,454 @@
+---
+sidebar_position: 3
+---
+
+# Solution
+
+## OKP4 Overview
+
+/!\ Info: this chapter is non-technical, suited to explain the concepts. The next chapter will dive into technical design.
```
Comment (ccamel) [inline docs/whitepaper/solution.md:9]: Either use an emoji (like ⚠️), or use an [admonition](https://docusaurus.io/docs/2.1.0/markdown-features/admonitions), for instance:

```md
:::caution

This chapter is non-technical, suited to explain the concepts. The next chapter will dive into technical design.

:::
```

File: docs/whitepaper/solution.md [hunk 2]
```diff
@@ -0,0 +1,454 @@
...
+
+Nevertheless, due to the complex and fragmented nature of our domain and its governance aspect, we need an architecture that is **more interoperable and expressive**. The OKP4 protocol has to allow any digital resources to **be understandable and interoperable** with the protocol. Furthermore, given the dynamic nature of the data and knowledge field, it's crucial that the OKP4 protocol possesses scalability to accommodate future growth and changes.
+
+For this, we need a specific design **(modularity)** to be able to interconnect and interpret resources of various nature **(interoperability)**
+
+![p2-05](/img/content/whitepaper/p2-05.webp)
+
+**What design for OKP4 Protocol ?**
+
+The OKP4 blockchain is a public delegated proof-of-stake layer 1 built with the Cosmos SDK.
```
Comment (ccamel) [inline docs/whitepaper/solution.md:101]:
```suggestion
The OKP4 blockchain is a public delegated proof-of-stake layer 1 built with the [Cosmos SDK](https://docs.cosmos.network/main).
```

File: docs/whitepaper/solution.md [hunk 3]
```diff
@@ -0,0 +1,454 @@
...
+
+*Cosmos in a nutshell:*
+
+*Cosmos is an application-specific blockchain (ASB)  ecosystem. It offers the ability for anyone to build their own sovereign ASB which is able to easily communicate with other blockchains. For its philosophy and its characteristics of sovereignty, modularity and interoperability, OKP4 is part of the Cosmos ecosystem.*
```
Comment (ccamel) [inline docs/whitepaper/solution.md:105]: Could use an admonition. Up to you.

File: docs/whitepaper/solution.md [hunk 4]
```diff
@@ -0,0 +1,454 @@
...
+
+**Cosmos SDK:** This open-source development kit allows the easy creation of a blockchain from a set of modules. In practice, **these modules** and their configurations provide specificity to the Protocol Layer. Several modules catering to generic functionalities, such as account management, staking, and token management, are provided within the Cosmos GitHub repository. This arrangement frees application developers to focus on designing modules that cater to their unique needs, instead of worrying about fulfilling the fundamental requirements of a blockchain.
+
+**IBC (Inter Blockchain Communication):** IBC is a general-purpose messaging protocol designed for communication between IBC-enabled blockchains. It's implemented through a specific module that ensures reliable, ordered, and authenticated communication. The IBC fosters an environment of interoperability, allowing distinct blockchains to interact seamlessly, thus enhancing the overall scalability and utility of the network.
```
Comment (ccamel) [inline docs/whitepaper/solution.md:113]:
```suggestion
**IBC (Inter Blockchain Communication):** [IBC](https://ibcprotocol.org/) is a general-purpose messaging protocol designed for communication between IBC-enabled blockchains. It's implemented through a specific module that ensures reliable, ordered, and authenticated communication. The IBC fosters an environment of interoperability, allowing distinct blockchains to interact seamlessly, thus enhancing the overall scalability and utility of the network.
```

File: docs/whitepaper/solution.md (hunk 1 above)
Comment (MathieuConte) [reply docs/whitepaper/solution.md:9]: Thanks !

**Review by tpelliet — APPROVED** [review]

**Pull request merged**